src/main/java/Duke.java

```diff
@@ -1,10 +1,129 @@
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    static ArrayList<Task> storage;
+    static ArrayList<String> instructions = new ArrayList<String>();
+    static int count = 0;
```

Comment:
Maybe a more explanatory name for count could be used to explain what this variable is counting?
```suggestion
    static int taskCount = 0;
```

Reply:
thanks!